Library/Homebrew/livecheck/strategy.rb

```diff
@@ -100,7 +100,8 @@ def self.page_headers(url)
             *args, url,
             print_stdout: false, print_stderr: false,
             debug: false, verbose: false,
-            user_agent: user_agent, retry: false
+            user_agent: user_agent, timeout: 20,
```

Comment:
Why `20` rather than the default? Explaining in a constant with a comment would be nice (and reusing it below)

Reply:
The default for `timeout` is `nil`, so a number has to be provided to enable it. This is 20 because it needs to be greater than the number we use for `--max-time` in `curl_args` (10 for `page_headers`, 15 for `page_content).

The latest commits add constants with comments and better aligns how `#page_headers` and `#page_content` interact with `Utils::Curl`.